query-engine/driver-adapters/executor/src/bench.ts

```diff
@@ -57,23 +64,37 @@ async function recordQueries(
   datamodel: string,
   prismaQueries: any
 ): Promise<void> {
-  const qe = await initQeWasmBaseLine(adapter, datamodel);
-  await qe.connect("");
+  // Different engines might have made different SQL queries to complete the same Prisma Query,
+  // so we record the results of all engines for the benchmarking phase.
+  const napi = await initQeNapiCurrent(adapter, datamodel);
+  await napi.connect("");
+  const wasmCurrent = await initQeWasmCurrent(adapter, datamodel);
+  await wasmCurrent.connect("");
+  const wasmBaseline = await initQeWasmBaseLine(adapter, datamodel);
+  await wasmBaseline.connect("");
+  const wasmLatest = await initQeWasmLatest(adapter, datamodel);
+  await wasmLatest.connect("");
```

Comment:
nit: Given that these `initQe*` have the same API (and all call the `.connect("")` method first and `.disconnect("")` method last), one could rationalise this code by mapping over the `[napi, wasmCurrent, wasmBaseline, wasmLatest]` before constructing the Query Engine instances.

However, this code could change in the future due to API differences between e.g. Napi and Wasm, so I'll approve this PR, leaving to @miguelff the judgement about whether to apply "DRY" or not in this case :)